docs/sources/user_guide/regressor/LinearRegression.ipynb

```diff
@@ -172,7 +172,8 @@
    "metadata": {},
    "source": [
     "- [1] Chapter 3, page 55, Linear Methods for Regression. Trevor Hastie; Robert Tibshirani; Jerome Friedman (2009). [The Elements of Statistical Learning: Data Mining, Inference, and Prediction (2nd ed.)](http://statweb.stanford.edu/~tibs/ElemStatLearn/download.html). New York: Springer. (ISBN 978–0–387–84858–7)\n",
-    "- [2] G. Strang, Linear Algebra and Its Applications, 2nd Ed., Orlando, FL, Academic Press, Inc., 1980, pp. 139-142."
+    "- [2] G. Strang, Linear Algebra and Its Applications, 2nd Ed., Orlando, FL, Academic Press, Inc., 1980, pp. 139-142.\n",
+    "- [3] [Ill-Conditioned Matrices](https://ece.uwaterloo.ca/~dwharder/NumericalAnalysis/04LinearAlgebra/illconditioned/)"
```

Comment:
It would be nice to modify this a bit so that readers know what kind of resource that is and who wrote it without having to click on it. I suggest the following modification:

- [3] Douglas Wilhelm Harder. *Numerical Analysis for Engineering*. Section 4.8, Ill-conditioned Matrices ([https://ece.uwaterloo.ca/~dwharder/NumericalAnalysis/04LinearAlgebra/illconditioned/](https://ece.uwaterloo.ca/~dwharder/NumericalAnalysis/04LinearAlgebra/illconditioned/)

Reply:
Okay

Reply:
Done